sbt-scalajs-bundler/src/main/scala/scalajsbundler/WebpackDevServer.scala

```diff
@@ -57,12 +52,9 @@ private [scalajsbundler] class WebpackDevServer {
 
     logger.info("Starting webpack-dev-server");
 
-    val devServerPath = npmDir / "node_modules" /
-    "webpack-dev-server" / "bin" / "webpack-dev-server.js"
-
     val command = Seq(
       "node",
-      devServerPath.absolutePath,
+      "node_modules/webpack-dev-server/bin/webpack-dev-server.js",
```

Comment:
Why not `workDir / "node_modules" / "webpack-dev-server" / "bin" / "webpack-dev-server.js"`?